users/validators.go

```diff
@@ -11,9 +12,9 @@ import (
 // Then, you can just call model.save() after the data is ready in DataModel.
 type UserModelValidator struct {
 	User struct {
-		Username string `form:"username" json:"username" binding:"exists,alphanum,min=4,max=255"`
-		Email    string `form:"email" json:"email" binding:"exists,email"`
-		Password string `form:"password" json:"password" binding:"exists,min=8,max=255"`
+		Username string `form:"username" json:"username" binding:"required,alphanum,min=4,max=255"`
+		Email    string `form:"email" json:"email" binding:"required,email"`
+		Password string `form:"password" json:"password" binding:"required,min=4,max=255"`
```

Comment:
Why need to change it? If it was needed, can we use a small change before the vote feature?

Reply:
Ditto for several place, is it related to postgreSQL?
https://github.com/gothinkster/golang-gin-realworld-example-app/pull/22/files#diff-dacc9ddedd396dc93f6f56d992c3cdfcR67
https://github.com/gothinkster/golang-gin-realworld-example-app/pull/22/files#diff-2c3aefbd8df6b8d84fee286ef54ba5dcR89